DESCRIPTION

```diff
@@ -81,7 +81,8 @@ Suggests:
     table1,
     gt,
     gtsummary
-Additional_repositories: https://mc-stan.org/r-packages/
+Remotes:
+   stan-dev/cmdstanr
```

Comment:
I'm sure the Additional_repositories way is what they need. Let's see if they can be more clear.... At least we could keep both.